### PR TITLE
add a use case for storing a data frame in a gist, #78

### PR DESCRIPTION
**Pull request by jsta**
See #78. I couldn't render the vignette as I was getting a Auth error on `gist('7ddb9810fc99c84c65ec')`.

### CODE REVIEW

File: inst/vign/gistr_vignette.Rmd [hunk 1]
```diff
@@ -296,3 +296,29 @@ gist_create(code = json, filename = "pelias_test.geojson")
...
+tc  <- textConnection('str', 'w', local = TRUE)
+write.csv(iris, file = tc, row.names = FALSE)
+close(tc)
+
+content <- list(content=paste(as.character(str), collapse='\n'))
+
+gistr::gist_create(code = {
+  content$content
+}, description = "using a gist as a data store", 
+filename = "iris.csv")
```
Comment (sckott) [inline inst/vign/gistr_vignette.Rmd]: can you put in the output here? so readers can see what it should look like? 

e.g. https://github.com/ropensci/gistr/blob/master/inst/vign/gistr_vignette.Rmd#L288-L293

File: inst/vign/gistr_vignette.Rmd [hunk 2]
```diff
@@ -296,3 +296,29 @@ gist_create(code = json, filename = "pelias_test.geojson")
...
+
+output <- read.csv(
+  text = gist(gists(what = "minepublic", per_page = 1)[[1]]$id)$
+    files$iris.csv$content)
+
+identical(output, iris)
```
Comment (sckott) [inline inst/vign/gistr_vignette.Rmd]: same here

**Comment by jsta** [comment]
Good idea. Latest commit shows expected output.

**Comment by sckott** [comment]
thx, all looks good!

**Pull request merged**